coordinator/points_writer.go

```diff
@@ -70,7 +70,17 @@ type PointsWriter struct {
 	}
 	subPoints chan<- *WritePointsRequest
 
-	statMap *expvar.Map
+	statMap struct {
+		NumWriteReq           stat.Int
+		NumPointWriteReq      stat.Int
+		NumPointWriteReqLocal stat.Int
+		NumWriteOK            stat.Int
+		NumWriteDropped       stat.Int
+		NumWriteTimeout       stat.Int
+		NumWriteErr           stat.Int
+		NumSubWriteOK         stat.Int
+		NumSubWriteDrop       stat.Int
+	}
```

Comment:
Two nits:

1) `Num` prefix doesn't go with our other naming. Generally we've used an `N` suffix to indicate counters (or just dropped any prefix/suffix entirely).

2) `statMap` could just be called `stats`. I don't think the `Map` part adds anything.